lib/Reader/reader.cpp

```diff
@@ -6994,6 +7001,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
     case ReaderBaseNS::CEE_LDC_I4_8:
     case ReaderBaseNS::CEE_LDC_I4_M1:
     LOAD_CONSTANT:
+      nop();
```

Comment:
This is to make sure that the store of the constant gets an ILOffset. What I was seeing is that when we have a LDC followed by a call, the call's ILOffset would be used instead of the load's. The EE expects both the call's offset (with the CALL_INSTRUCTION sourcetype) and the load's offset because the stack is empty. Generally speaking, LDC's are at locations where the stack is empty, and the debugger expects il offsets for all points where the stack is empty. I could add a comment to make that clear.